lib/list.js

```diff
@@ -152,7 +153,7 @@ ListDOMPatcher.prototype = {
 					this.move,
 					[patch.toIndex, patch.fromIndex]
 				);
-			} else {
+			}	else if (patch.type === "splice") {
```

Comment:
is there a tab character between `}` and `else`

Reply:
You were right! Thanks for the heads up. The changes have been pushed up.